peepingtom/datablocks/simpleblocks/propertyblock.py

```diff
@@ -26,6 +30,19 @@ def items(self):
         for k, v in self.data.items():
             yield k, np.array(v)
 
+    def __setitem__(self, key, value):
+        # does not work on view (we lose track of how nested we are and how we got here)
+        if self.is_view():
+            raise TypeError('cannot set data in a view. Use the parent datablock.')
```

Comment:
might be worth pointing out how to access the parent in this message?

Reply:
Make sense, but I'm not sure how to best phrase this. This will happen if one does:
```
datablock[x][y] = 1
```
there is no trivial way to do the same thing with a single getitem call. This is also a pandas issue, not just us. How would you point this out? Just saying "use datablock.view_of" is not really enough, I think...

Reply:
Fair, I can't immediately think of a good way to solve that... I guess the real solution is settable views but that's more work of course 😆

Reply:
For now, I'll just remove the reference to `parent` from the error message, and make it clearer that this just won't work. It might be a task for the future :)